modules/tabs-v2/assets/scss/frontend.scss

```diff
@@ -46,6 +46,7 @@
 	--tabs-v2-title-border-radius: initial;
 	--tabs-v2-title-transition: 0.3s;
 
+  	--tabs-v2-icon-vertical-align: center;
```

Comment:
Please use the flex box methods for the naming of css variables, in this case possibly `--tabs-v2-icon-align-items`